docs/workflow/orchestration.md

```
@@ -116,43 +69,37 @@ Process:
 4. Output: Displays results
 ```
 
-### 3. Metric-to-SQL Workflow
+### 2. Gen SQL Agentic Workflow
 
-**Purpose**: Generate SQL from predefined business metrics
+**Purpose**: Generate SQL with database, semantic-layer, metric-search, and other configured tools.
 
 **Node Sequence:**
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)